doc/SONIC_Test_Ingress_Discard_All_HLD.md

```diff
@@ -0,0 +1,816 @@
+- Execute CLI command - ```show interface counters rif```
+- Verify ```RX_ERR``` incremented
+---
+- PTF host will send packet2
+- When packet reaches SONIC DUT, it should be dropped according to the test objective
+- Execute CLI command - ```show interface counters rif```
+- Verify ```RX_ERR``` incremented
+
+#### Test case #16
+##### Test objective
+
+Verify DUT drops packet where SRC IP address is not specified
+
+IPv4 sip == 0.0.0.0/32
+
+Note: for IPv6 (sip == ::0)
+
+Packet1 to trigger drop
+```
+...
+###[ IP ]###
+    version = 4
+    ttl = [auto]
+    proto = tcp
+    src = [0.0.0.0]
+    dst = [auto]
+...
+```
+
+Packet2 to trigger drop
+```
+...
+###[ IP ]###
+    version = 6
+    ttl = [auto]
+    proto = tcp
+    src = [::0]
+    dst = [auto]
+...
+```
+
+##### Get interfaces which are members of LAG, RIF and VLAN. Repeat defined test steps for each of those interfaces.
+
+##### Test steps
+- PTF host will send packet1
+- When packet reaches SONIC DUT, it should be dropped according to the test objective
+- Execute CLI command - ```show interface counters rif```
+- Verify ```RX_ERR``` incremented
+---
+- PTF host will send packet2
+- When packet reaches SONIC DUT, it should be dropped according to the test objective
+- Execute CLI command - ```show interface counters rif```
+- Verify ```RX_ERR``` incremented
+
+#### Test case #17
+##### Test objective
+
+Verify DUT drops packet where DST IP address is not specified
+
+IPv4 sip == 0.0.0.0/32
+
+Note: for IPv6 (sip == ::0)
+
+Packet1 to trigger drop
+```
+...
+###[ IP ]###
+    version = 4
+    ttl = [auto]
+    proto = tcp
+    src = [auto]
+    dst = [0.0.0.0]
+...
+```
+
+Packet2 to trigger drop
+```
+...
+###[ IP ]###
+    version = 6
+    ttl = [auto]
+    proto = tcp
+    src = [auto]
+    dst = [::0]
+...
+```
+
+##### Get interfaces which are members of LAG, RIF and VLAN. Repeat defined test steps for each of those interfaces.
+
+##### Test steps
+- PTF host will send packet1
+- When packet reaches SONIC DUT, it should be dropped according to the test objective
+- Execute CLI command - ```show interface counters rif```
+- Verify ```RX_ERR``` incremented
+---
+- PTF host will send packet2
+- When packet reaches SONIC DUT, it should be dropped according to the test objective
+- Execute CLI command - ```show interface counters rif```
+- Verify ```RX_ERR``` incremented
+
+#### Test case #18
+##### Test objective
+
+Verify DUT drops packet when configured ACL DROP for SRC IP 20.0.0.0/24
+
+Packet1 to trigger drop
+```
+...
+###[ IP ]###
+    version = 4
+    ttl = [auto]
+    proto = tcp
+    src = [20.0.0.10]
+    dst = [auto]
+...
+```
+
+##### Get interfaces which are members of LAG, RIF and VLAN. Repeat defined test steps for each of those interfaces.
+
+##### Test steps
+- PTF host will send packet1
+- When packet reaches SONIC DUT, it should be dropped according to the test objective
+- Execute CLI command - ```aclshow -a```
```

Comment:
Should we be checking `show interface counters` as well?

Reply:
Added.